package.json

```diff
@@ -45,7 +45,7 @@
 		"lint": "lerna run lint",
 		"lint:fix": "lerna run lint:fix",
 		"test": "lerna run test",
-		"test:elements": "lerna run test --ignore=lisk-commander --ignore=lisk-framework --ignore=lisk-sdk --ignore=@liskhq/lisk-framework*",
+		"test:elements": "lerna run test --ignore=lisk-commander --ignore=lisk-framework --ignore=lisk-sdk --ignore=@liskhq/lisk-framework* -- -u",
```

Comment:
Are we sure we want to update snapshot on each run? Maybe it would be better not to in case the need to update snapshots is an unwanted side effect?

Reply:
i forgot to remove 😓